src/plone/api/tests/test_content.py

```diff
@@ -1042,7 +1042,7 @@ def test_find_interface_dict__include_not_query(self):
             },
         )
 
-        self.assertEqual(len(brains_all) - len(brains), 1)
+        self.assertEqual(len(brains_all) - len(brains), 0)
```

Comment:
The assert here should be 2. You can see the error in the test here:

https://jenkins.plone.org/job/pull-request-6.1-3.11/298/testReport/junit/plone.api.tests.test_content/TestPloneApiContent/test_find_interface_dict__include_not_query/

```suggestion
         # Plone Site also implements the IContentish and INavigationRoot interfaces.
        self.assertEqual(len(brains_all) - len(brains), 2)
```